distribution/src/bin/elasticsearch-service.bat

```diff
@@ -34,6 +34,7 @@ if /i %SERVICE_CMD% == start goto doStart
 if /i %SERVICE_CMD% == stop goto doStop
 if /i %SERVICE_CMD% == manager goto doManagment
 echo Unknown option "%SERVICE_CMD%"
+exit /B 1
```

Comment:
What's the motivation behind `/B` here?

>    /B        When used in a batch script, this option will exit 
              only the script (or subroutine) but not CMD.EXE
              If executed from outside a batch script, if will quit CMD.exe

Reply:
I copied what we had for existing exit calls. Before we would exist 0 even though there was an error in the cases I have added explicit exit.